README.md

```diff
@@ -130,6 +130,22 @@ In this example content is posted along with the recipe. Where the payload body
 }
 ```
 
+It's also possible to define random content inside the posted body. This can be useful to
+simulate JSON contracts, structured text, etc.
+
+```json
+{
+    "body": "{\"data\":\"<<256kb>>\", \"metadata\":\"<<128b>>and<<16b>>\"}",
```

Comment:
Why would we want to have two different random values inside one string? I can see why we would want different random strings in different places inside one payload.

Reply:
I would have thought that would be quite a common use case? In the case of JSON response for example with multiple keys that need values are we ever likely only to want to randomise one part of the response body? 🤔

Reply:
No. When I said one string, I meant one value. It's a bit odd to have two random options inside one single value inside a payload. Having many keys and values I can see why we would want different values belonging to different keys to have different sizes.

Reply:
ahhh I think I understand what you mean. I agree it's unlikely a single json key value is likely to need multiple random section, but you have to remember that these response bodies are just being treated as dumb strings. Origin simulator has no concept that it's a JSON object, and it simply substituting all <<*kb>> with random characters

Reply:
And besides:
`"{\"data\": [ \"<<128b>>\", \"<<128b>>\", \"<<128b>>\"}"`
Seems like a potential use case where more than one random section per json value would be useful